lib/usagereporter/web/userevent_discover_test.go

```diff
@@ -114,6 +114,31 @@ func TestDiscoverEventDataToUsageEvent(t *testing.T) {
 				},
 			}},
 		},
+		{
+			name:     uiDiscoverDeployServiceEvent + "/success_test_deployed_method_setting",
```

Comment:
That's one of the two "deployed method" that `rg "deployed.?method"` has returned. ;)

```suggestion
			name:     uiDiscoverDeployServiceEvent + "/success_test_deploy_method_setting",
```